gensim/scripts/make_wiki.py

```diff
@@ -1 +0,0 @@
-make_wikicorpus.py
```

Comment:
I merge all conflicts manually, using Sublime, and then commit my changes. I have not come across any wiki related merge conflicts, so it happened automatically? The other space related ones are probably me, because of the manual merge conflict removal, but they seem harmless.